rhino.yml

```diff
@@ -0,0 +1,195 @@
+version: '2'
+
+services:
+################################################################################
+#
+# the CAN Interfaces for Rhino need an odsupercomponent.
+#
+################################################################################
+
+    odsupercomponent:
+        image: seresearch/opendlv-rhino-on-opendlv-on-opendlv-core-on-opendavinci-on-base:v0.1.5
+        restart: on-failure
+        network_mode: "host"
+        volumes:
+        - .:/opt/opendlv.data
+        command: "/opt/od4/bin/odsupercomponent --configuration=/opt/opendlv.data/configuration --cid=111"
+
+######################################################################################
+#
+# Microservices for RHINO (Volvo FH16).
+#
+######################################################################################
+
+    # OxTS GPS
+    dev-gps-ncom:
+        image: chalmersrevere/opendlv-device-gps-ncom-multi:v0.0.15
+        restart: on-failure
+        network_mode: "host"
+        command: "--ncom_ip=0.0.0.0 --ncom_port=3000 --cid=111"
+
+    # Velodyne VLP32c Lidar
+    dev-lidar-vlp32c:
+       image: chalmersrevere/opendlv-device-lidar-vlp32c-multi:v0.0.2
+       restart: always
+       network_mode: "host"
+       command: "--vlp32c_ip=0.0.0.0 --vlp32c_port=2368 --cid=111"
+
+    # Accessing a video stream using OpenCV.
+#    dev-camera-opencv:
+#        image: chalmersrevere/opendlv-device-camera-opencv-multi:v0.0.10
+#        restart: always
+#        ipc: "host"
+#        network_mode: "host"
+#        volumes:
+#        - /tmp:/tmp
+#        command: "--camera=http://root:axiscamera@10.40.40.29/axis-cgi/mjpg/video.cgi\\?channel=0\\&.mjpg --width=640 --height=480 --freq=20 --name.i420=video0.i420 --name.argb=video0.argb"
+
+    # Encoder for camera-opencv(x264)
+    # x264(for research) has the better performance;
+#    video-x264-encoder-amd64:
+#        build:
+#            context: https://github.com/chalmers-revere/opendlv-video-x264-encoder.git
+#            dockerfile: Dockerfile.amd64
+#        restart: on-failure
+#        network_mode: "host"
+#        ipc: "host"
+#        volumes:
+#        - /tmp:/tmp
+#        command: "--cid=111 --name=video0.i420 --width=640 --height=480"
+
+    # Volvo Angle Sensor using lab jack
+#    dev-io-ue9-amd64:

```

Comment:
I tested this, it's the old truck angle sensor which still works with the new angle sensor. However, it's on a private branch i think i.e. not an official OpenDLV microservice. It might become obsolete in a few months, but since it works now, i've added it.

Reply:
Okay for me.